completions/zsh/CMakeLists.txt

```diff
@@ -1,9 +1,8 @@
 set(ZSH_COMPLETION_DIR "${CMAKE_INSTALL_FULL_DATADIR}/zsh/site-functions")
 
-configure_file(adb.in _adb)
 configure_file(fastboot.in _fastboot)
```

Comment:
The fastboot completion file is not subject to the problem described in https://github.com/nmeum/android-tools/issues/38#issuecomment-855372692? For example, the need for the extra `bashcompinit` configuration et cetera.